src/cases/var_io_G_case_scorpio.cpp

```diff
@@ -251,7 +251,7 @@ int run_varn_G_case_scorpio (e3sm_io_config &cfg,
     int i, j, k, err, rank, ncid;
     e3sm_io_scorpio_var *varids;
     int scorpiovars[7];
-    int rec_no=0, my_nreqs, *nvars_D=cfg.nvars_D;
+    int rec_no = -1, my_nreqs, nvars_D[6];
```

Comment:
@khou2020 
This change at line 254 for nvars_D should be reversed.
`cfg.nvars_D[]` will be used in `report_timing_WR()`

Reply:
fixed